### DISCUSSION TITLE
New version compatible with DRF 3.13 ?

### DISCUSSION THREAD

**Discussion by nikosmichas** [django-json-api/django-rest-framework-json-api#1037] [General]
Hello :wave:

I see that [this commit](https://github.com/django-json-api/django-rest-framework-json-api/commit/fb8f6c465b9aecf2a40b5dba58b89a52397ad2a1) enables installing with DRF >= 3.13 

Is a new release expected soon with this change?

**Comment by sliverc** [comment]
When releasing DJA 4.3.0 I counted on DRF releasing Django 4.0 support in a point release. Unfortunately there was a breaking change after all so 3.13 was needed. As planning of DJA 5.0.0 goes two things are missing which is dropping of EOL Python 3.6 and removing of deprecation. This should hopefully be ready in January.

This said for people who stumble upon this discussion: DRF 3.13 only adds support for Django 4.0. If this is not needed staying with DRF 3.12 for now will do just as well.

**Reply by sliverc** [reply]
Version 5.0.0 with DRF 3.13 support has been released.

**Reply by nikosmichas** [reply]
Great, thanks a lot!
